src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import core.basesyntax.service.GenerateReport;
+import core.basesyntax.service.ReadDataFile;
```

Comment:
Ensure that the `ReadDataFile` class is correctly named and placed in the appropriate package as per the task requirements.